indexer/ingests_indexer/src/test/scala/uk/ac/wellcome/platform/archive/indexer/ingests/IngestIndexerTest.scala

```diff
@@ -99,4 +101,137 @@ class IngestIndexerTest
       _ shouldBe a[JavaClientExceptionWrapper]
     }
   }
+
+  describe("orders updates correctly") {
+    describe("when both ingests have a modified date") {
+      val ingestId = createIngestID
+
+      val olderIngest = createIngestWith(
+        id = ingestId,
+        events = Seq(
+          IngestEvent(description = "event 1", createdDate = Instant.ofEpochSecond(101))
+        ),
+        createdDate = Instant.ofEpochSecond(1)
```

Comment:
Not sure if you want to update the tests too - shouldn't make a difference, but might be more self-documenting.